README.md

```diff
@@ -39,29 +39,30 @@ The gem is available as open source under the terms of the [MIT License](https:/
 # Usage
 
 ### Available Records
-* Lockstep::Contact
-* Lockstep::Connection
 * Lockstep::Account
-* Lockstep::User
-* Lockstep::Note
```

Comment:
Why do we have so many changes here ?

Reply:
Reordered the README.md to make the available models in alphabetical order. Looks presentable imo 😄